mojave/items/clothing/under/ncr/under.dm

```diff
@@ -5,18 +5,18 @@
 	name = "NCR fatigues"
 	desc = "A uniform bearing the marks of the New California Republic."
 	icon_state = "ncr_uniform"
-	inhand_icon_state = "ncr_uniform"
-
-/obj/item/clothing/under/ms13/ncr/fatigues/winter
-	name = "NCR winter fatigues"
-	desc = "A winter uniform bearing the marks of the New California Republic."
-	icon_state = "ncr_uniform_w"
 	inhand_icon_state = "ncr_uniform_w"
 
-/obj/item/clothing/under/ms13/ncr/fatigues/officer
-	name = "NCR winter officer fatigues"
-	desc = "An NCR uniform normally worn by NCR officers, intended for winter operations."
-	icon_state = "ncr_o_w"
+/obj/item/clothing/under/ms13/ncr/fatigues/scout
```

Comment:
What are these scout fatigues meant to be for? What do they even look like in comparison?

Reply:
I forgot I left those in. Whoops. They don't look too bad, though I didn't want them to be in in the first place.  I think they're from memager.

Reply:
Ultimately I guess it honestly doesn't matter if they're in or not if we just don't use them. I was mostly just confused what they looked like and what they were intended for.

Reply:
![image](https://user-images.githubusercontent.com/51188571/106324973-10906d00-6248-11eb-982d-879e6e37e93c.png)

There's really not any purpose in it currently. at the time I just thought to code it in so it could be used for... Whatever someone wanted at some point. This is what they look like, boots included.

Reply:
Quirky.